src/v2/Apps/Order/OrderApp.tsx

```diff
@@ -37,6 +37,12 @@ export const preventHardReload = event => {
   event.returnValue = true
 }
 
+export enum PaymentMethods {
+  CreditCard = "credit_card",
+  BankDebit = "bank_debit",
+  WireTransfer = "wire_transfer",
+}
+
```

Comment:
moved this enum to OrderApp as it seemed the appropriate central place, since both payment and review steps will use this enum.